codemods/1.x-2.x/__testfixtures__/operators-flow.output.js

```diff
@@ -14,7 +14,5 @@ export default [
   },
 
   // Wait 200ms, then continue chain
-  wait(200, [
-    doSomething
-  ])
+  wait(200)
```

Comment:
This one is proving hard. Moving the array starts messing with the whole piece of code's whitespace.